client/src/pages/UsersPage.js

```diff
@@ -20,30 +20,32 @@ export default function UsersPage({ users, title }) {
   return (
     <Wrapper>
       <Header>{title}</Header>
-      {users.map(({ id, name, phone, email, department, skills }) => (
-        <SingleEntry key={id}>
-          <User
-            name={name}
-            phone={phone}
-            mail={email}
-            department={department}
-            skills={skills}
-          />
-        </SingleEntry>
-      ))}
+      <ListWrapper>
+        {users.map(({ id, skills, name, phone, email, department }) => (
+          <SingleEntry key={id}>
+            <User
+              name={name}
+              phone={phone}
+              mail={email}
+              department={department}
+              skills={skills}
+            />
+          </SingleEntry>
+        ))}
+      </ListWrapper>
```

Comment:
From the naming I would think that `ListWrapper` is a list. Now you create a list for every user 🤔 Should probably be that `ListWrapper` is an `ul` and `SingleEntry` an `li`. That would be the correct markup. Also, `User` should then be a `section`.